submissions/chris-voitova/oop/index.js

```diff
@@ -0,0 +1,110 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: _put repo URL here_
+   Web app: _put project's github pages URL here_
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+
+// ======== OUTPUT ========
+/* Use print(message) for output.
+   Default tag for message is <pre>. Use print(message,'div') to change containing element tag.
+
+   Message can contain HTML markup. You may also tweak index.html and/or styles.css.
+   However, please, REFRAIN from improving visuals at least until your code is reviewed
+   so code reviewers might focus on a single file that is index.js.
+   */
+
+/* Print examples:
+   print('ABC');
+   print('<strong>ABC</strong>');
+   print('<strong>ABC</strong>', 'div');
+
+   print('human; John; male; 2; 2; Hello world!; Rex, Tom, Jenny');
+   print('human; <strong>John</strong>; male; 2; 2; <em>Hello world!</em>; Rex, Tom, Jenny');
+   print('human; <strong>John</strong>; male; 2; 2; <em>Hello world!</em>; Rex, Tom, Jenny', 'div');
+   */
+
+class Inhabitant {
+  constructor({ name, greating }) {
+    this.name = name;
+    this.greating = greating;
+  }
+  saying() {
+    return this.greating;
+  }
+  toString() {
+    let props = Object.entries(this);
+    props = props.map(([key, value]) => {
+      return value && key !== "greating" ? `${key}: ${value}` : null;
+    });
```

Comment:
It can be simplified even further, but it doesn't really matter.

```javascript
props = props.map(([key, value]) => value && key !== "greating" ? `${key}: ${value};` : null);
```